source/help/getting-started/organizing-your-sidebar.rst

```diff
@@ -0,0 +1,211 @@
+
+    .. image:: ../../images/multi-select-move.gif
+
+Muting and unmuting categories
+------------------------------
+
+When you mute or unmute a category, all channels within that category are also muted or unmuted. You can still selectively unmute specific channels within a muted category.
+
+Select the **Category options** icon in the sidebar, then select **Mute Category**.
+
+Once a category is muted:
+
+- Email, desktop, and push notifications are disabled for all channels in the category.
+- A mute icon displays next to each channel name in the category.
+- The category and all of its channels appear at reduced opacity in the left-hand sidebar. Channels in the category aren't marked as unread unless you’re mentioned directly.
+
+To unmute the category, select the **Category options** icon in the sidebar, then select **Unmute Category**.
+
+    .. image:: ../../images/mute-categories.gif
+
+Sorting channels in categories
+------------------------------
+
+Select the **Category options** icon in the sidebar, then select **Sort** and choose from **Alphabetically**, **Recent Activity**, or **Manually**.
+
+    .. image:: ../../images/sort-categories.gif
+
+Grouping unread messages
+------------------------
+
+Catch up on all your unread channels in one place at the top of your sidebar with a one-click **Unreads** category. By default, Mattermost offers a one-click **Unreads** filter to only show channels with unread activity. Instead, you may choose to automatically group unread channels in their own category at the top of your sidebar.
+
+Go to **Main Menu > Account Settings > Sidebar**, set **Group unread channels separately** to **On**, then select **Save**.
+
+- When this setting is enabled, all unread messages appear only in the **Unreads** category.
+- When this setting is disabled, all unread messages appear within their respective categories and channels.
+
+When enabled, the **Unreads** category is organized as follows:
+
+- Unread messages that contain mentions are sorted to the top by most recent activity.
+- Unread messages that do not contain mentions are sorted by most recent and appear directly below unread messages containing mentions.
+- Muted channels that contain mentions are sorted by most recent and appear directly below unread messages without mentions.
+
+    .. image:: ../../images/unreads.gif
+
+.. tip::
+  If you prefer to see a decicated unread-only view in your sidebar, collapse all custom categories to show only unread messages, then disable **Grouping unread messages** under **Account Settings > Sidebar**.
+
+Sorting your Direct Messages
+----------------------------
+
+Sort your Direct Messages alphabetically or by recent conversations first.
+
+Select the **Channel options** icon in the sidebar, then select **Sort** and choose from **Alphabetically** or **Recent Activity**.
+
+How many Direct Messages to display?
```

Comment:
Perhaps go a step down on heading for this one and change the heading of the section above to Managing Direct Messages. 0/5